### PR TITLE
Update bval 11 fat to run tests with bval 20

### PR DESCRIPTION
**Pull request by awisniew90**
Update bvall 11 fat to additionally run tests with bval 20

### CODE REVIEW

File: dev/com.ibm.ws.beanvalidation.v11_fat/.classpath [hunk 1]
```diff
@@ -14,5 +14,6 @@
 	<classpathentry kind="src" path="test-applications/DefaultBeanValidationCDI_11.war/src"/>
 	<classpathentry kind="con" path="aQute.bnd.classpath.container"/>
 	<classpathentry kind="con" path="org.eclipse.jdt.launching.JRE_CONTAINER/org.eclipse.jdt.internal.debug.ui.launcher.StandardVMType/JavaSE-1.8"/>
+	<classpathentry kind="src" path="/fattest.simplicity"/>
```
Comment (aguibert) [inline dev/com.ibm.ws.beanvalidation.v11_fat/.classpath]: this line can be removed

File: dev/com.ibm.ws.beanvalidation.v11_fat/fat/src/com/ibm/ws/beanvalidation/fat/tests/BasicValidation10CommonTest.java [hunk 2]
```diff
@@ -196,6 +198,7 @@ public void testCustomConstraintValidatorFactory10() throws Exception {
      */
     @Test
     public void testBuildApacheConfiguredValidatorFactory10() throws Exception {
+        assumeTrue(bvalVersion < 20);
```
Comment (aguibert) [inline dev/com.ibm.ws.beanvalidation.v11_fat/fat/src/com/ibm/ws/beanvalidation/fat/tests/BasicValidation10CommonTest.java]: Never used `assumeTrue()` before.  If the assumption is false, does the test get reported as skipped?

File: dev/com.ibm.ws.beanvalidation.v11_fat/publish/servers/com.ibm.ws.beanvalidation.cdi_2.0.fat/.gitignore [hunk 3]
```diff
@@ -0,0 +1,3 @@
+/derby
```
Comment (aguibert) [inline dev/com.ibm.ws.beanvalidation.v11_fat/publish/servers/com.ibm.ws.beanvalidation.cdi_2.0.fat/.gitignore]: this file can be removed

File: dev/com.ibm.ws.beanvalidation.v11_fat/publish/servers/com.ibm.ws.beanvalidation_2.0.fat/.gitignore [hunk 4]
```diff
@@ -0,0 +1,3 @@
+/derby
```
Comment (aguibert) [inline dev/com.ibm.ws.beanvalidation.v11_fat/publish/servers/com.ibm.ws.beanvalidation_2.0.fat/.gitignore]: this file can be removed

File: dev/com.ibm.ws.beanvalidation.v11_fat/fat/src/com/ibm/ws/beanvalidation/fat/tests/BasicValidation10CommonTest.java (hunk 2 above)
Comment (awisniew90) [reply dev/com.ibm.ws.beanvalidation.v11_fat/fat/src/com/ibm/ws/beanvalidation/fat/tests/BasicValidation10CommonTest.java]: Yup. False marks the test skipped.

**Comment by awisniew90** [comment]
#build

**Pull request merged**